server/schedule/operator/step.go

```diff
@@ -36,7 +35,7 @@ type OpStep interface {
 	fmt.Stringer
 	ConfVerChanged(region *core.RegionInfo) uint64
 	IsFinish(region *core.RegionInfo) bool
-	CheckInProgress(cluster opt.Cluster, region *core.RegionInfo) error
+	CheckInProgress(checkFn func(id uint64) error, region *core.RegionInfo) error
```

Comment:
`checkFn` is not easy to understand. maybe define a `type CheckStoreStateFn func(id uint64) error`.